content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -1,5 +1,278 @@
+potentially other facets of isolation - the same things that isolate a Docker
+container.  Within a Pod's context, the individual applications may have
+further sub-isolations applied.
+
+In terms of Docker concepts, a Pod is similar to a group of Docker containers
+with shared namespaces and shared filesystem volumes.
+
+## Using Pods
+
+Usually you dont' need to create Pods directly. Instead, you can almost
+always use workload resources such as {{< glossary_tooltip text="Deployment"
+term_id="deployment" >}} or {{< glossary_tooltip text="Job" term_id="job" >}},
+even for singleton Pods.
+If your Pods need to track state, there is a
+{{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}} resource intended
+to be the best fit for that use case.
+
+Pods in a Kubernetes cluster can be used in two main ways:
+
+* **Pods that run a single container**. The "one-container-per-Pod" model is the
+  most common Kubernetes use case; in this case, you can think of a Pod as a
+  wrapper around a single container; Kubernetes manages Pods rather than managing
+  the containers directly.
+* **Pods that run multiple containers that need to work together**. A Pod might
+  encapsulate an application composed of multiple co-located containers that are
+  tightly coupled and need to share resources. These co-located containers might
+  form a single cohesive unit of service—one container serving data stored in a
+  shared volume to the public, while a separate _sidecar_ container refreshes or
+  updates those files.  
+  The Pod wraps these containers, storage resources, and an ephemeral network
+  identity together as a single unit.
+
+  {{< note >}}
+  Grouping multiple co-located and co-managed containers in a single Pod is a
+  relatively advanced use case. You should use this pattern only in specific
+  instances in which your containers are tightly coupled.
+  {{< /note >}}
+
+Each Pod is meant to run a single instance of a given application. If you want to
+scale your application horizontally (to provide more overall resources by running
+more instances), you should use multiple Pods, one for each instance. In
+Kubernetes, this is typically referred to as _replication_.
+Replicated Pods are usually created and managed as a group by a workload resource
+and its {{< glossary_tooltip text="controller" term_id="controller" >}}.
+
+See [Pods and controllers](#pods-and-controllers) for more information on how
+Kubernetes uses workload resources, and their controllers, to implement application
+scaling and auto-healing.
+
+### How Pods manage multiple containers
+
+Pods are designed to support multiple cooperating processes (as containers) that form
+a cohesive unit of service. The containers in a Pod are automatically co-located and
+co-scheduled on the same physical or virtual machine in the cluster. The containers
+can share resources and dependencies, communicate with one another, and coordinate
```

Comment:
```suggestion
share resources and dependencies, communicate with one another, and coordinate
```

Reply:
I think the original text is more accurate. “These containers coordinate when and how they are terminate” is usually _not_ true.